### ISSUE TITLE
Setting USB-VCP to OFF is not persistent

### ISSUE THREAD

**Issue by mha1** [EdgeTX/edgetx#1952]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

2.7.1: Setting USB-VCP to OFF doesn't survive radio life cycle. USB-VCP is set back to CLI in next life cycle.

This is probably also the reason why it seems as if USB-VCP defaults to CLI while (I think) it should default to OFF, shouldn't it?

### Expected Behavior

USB-VCP set to OFF is stored persistently

### Steps To Reproduce

Go to Radio Settings/Hardware
Set USB-VCP from CLI to OFF
Leave Hardware Tab
Turn off radio
Turn on radio
Go to Radio Settings/Hardware
Observe USB-VCP setting is back to CLI

### Version

2.7.0

### Transmitter

Radiomaster TX16S

### Anything else?

initial radio.yml excerpt with USB-VCP set to CLI
![image](https://user-images.githubusercontent.com/5615068/167829641-931b1bd0-94b6-48f7-8704-5b9a5d2c41ec.png)

radio.yml excerpt after setting USB-VCP set to OFF before turning radio off
![image](https://user-images.githubusercontent.com/5615068/167829788-9072d28b-d006-4ae0-8e01-be4a75866391.png)

radio.yml excerpt after turning radio on again
![image](https://user-images.githubusercontent.com/5615068/167829857-6a3cd17f-a5aa-469d-9040-6fbb7105de4a.png)

**Comment by pfeerick** [comment]
Transmitters that support ELRS passthrough (i.e. TX16S and Zorro) will always reset USB-VCP being set to CLI if it is set to OFF, as failing to do so will prevent passthrough updates from functioning, as it depends on CLI being active. This was a deliberate choice, so is by design, not a bug. I guess it could be improved slightly by also checking for whether the internal module is CRSF or not, but a future option being considered is be being able to also do pass-though update for the MPM.

**Comment by mha1** [comment]
Thanks for the explanation (which is not obvious :-)). Works for me.

**Comment by rotorman** [comment]
I wonder if it might be better to remove USB-VCP Option "OFF" from the menu of radios that support pass-through flashing of internal Express LRS in order to make this more intuitive/clearer.

**Comment by mha1** [comment]
more intuitive -> more good :-)

**Comment by pfeerick** [comment]
Yeah, that probably is the way to go... yes, it *does* mean if the VCP is set to anything other than CLI that passthrough will fail, but that is the case now anyway, so is more of a user education problem. I think it should be specifically tied to the internal module selected, and is easy to filter out on colorlcd, and B&W isn't that much harder. But companion needs to know about this also.

**Comment by rotorman** [comment]
@mha1 could you please test #1957

**Comment by mha1** [comment]
I have checked and am not sure if I understood the change correctly. Trying this on my TX16s with internal module set to MULTI. I can't set USB-VCP to CLI anymore. But when I set it to OFF it'll be back to CLI again in the next life cycle.

![image](https://user-images.githubusercontent.com/5615068/168325727-181fc6e3-6e47-4e0a-8afc-656a8b10bd8f.png)
